lib/tasks/multitenant/triggers.rake

```diff
@@ -38,6 +38,225 @@ namespace :multitenant do
     puts "Recreated #{triggers.size} triggers"
   end
 
+  desc 'Fix empty or corrupted tenant_id in accounts'
+  task :fix_corrupted_tenant_id_accounts, %i[batch_size sleep_time] => :environment do |_task, args|
+    batch_size = (args[:batch_size] || 100).to_i
+    sleep_time = (args[:sleep_time] || 1).to_i
+
+    ids = [2445582551393, 2445582551394, 2445582551395, 2445582551396, 2445582551397, 2445582574326, 2445582574327,
```

Comment:
What are those ids?
We should not have them in the code as this is OSS

Reply:
For this:
https://github.com/3scale/infrastructure/issues/693#issuecomment-496268189